### PR TITLE
net-irc/inspircd: Clean-up and a fix

### PR DESCRIPTION
**Pull request by clinew**

### CODE REVIEW

File: net-irc/inspircd/inspircd-2.0.25.ebuild [hunk 1]
```diff
@@ -20,7 +20,7 @@ RDEPEND="
 	geoip? ( dev-libs/geoip )
```
Comment (mgorny) [inline net-irc/inspircd/inspircd-2.0.25.ebuild]: Any reason not to `Closes:` that bug instead?

Comment (clinew) [reply net-irc/inspircd/inspircd-2.0.25.ebuild]: Nope, changed.

**Comment by clinew** [comment]
Updated!

File: net-irc/inspircd/inspircd-2.0.25.ebuild [hunk 2]
```diff
@@ -20,7 +20,7 @@ RDEPEND="
...
 	gnutls? ( net-libs/gnutls:= dev-libs/libgcrypt:0 )
 	ldap? ( net-nds/openldap )
-	mysql? ( virtual/mysql )
+	mysql? ( dev-db/mysql-connector-c )
```
Comment (mgorny) [inline net-irc/inspircd/inspircd-2.0.25.ebuild]: `:=`?

**Comment by mgorny** [comment]
Feel free to just ACK instead of updating, and I'll fix it while merging.

File: net-irc/inspircd/inspircd-2.0.25.ebuild (hunk 2 above)
Comment (clinew) [reply net-irc/inspircd/inspircd-2.0.25.ebuild]: Not sure what I'd be `:=`-ing, the instructions read: "If you need libmysqlclient.so, please depend on dev-db/mysql-connector-c. If you need or can use libmariadb.so, please depend on
dev-db/mariadb-connector-c."

**Comment by clinew** [comment]
ACK

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**